ACHNBrowserUI/ACHNBrowserUI/views/items/ItemsListView.swift

```diff
@@ -73,6 +86,7 @@ struct ItemsListView: View {
                     }
                 }
             }
+            .id(viewModel.sort)
```

Comment:
SwiftUI struggles with swapping large lists such as what occurs when sorting. By setting the List's id to the sort, we can tell SwiftUI to totally redraw the List instead of trying to diff all of the items. This improves the experience quite a bit.

Reply:
Yup. I saw it happen, first I thought it was Swift being really slow at sorted. And then I understood it was the List animation. I've tried the .id(UUID()) method but it break the search. So this one is a clever one!

Reply:
Yeah, this way it only changes majorly when a major change occurs :)